test-three/amy-menu/src/menu.tsx

```diff
@@ -0,0 +1,93 @@
+import { Component } from "react";
+import menuJson from "./menu-data.json";
+
+type MenuItemProps = {
+  name: string;
+};
+
+class MenuSubItem extends Component<MenuItemProps> {
+  render() {
```

Comment:
Is there a reason you chose to use a stateful component in here? Function/Functional components are a lot less heavy on the front end

Reply:
I assumed that a stateful component was necessary to access props. Good to know about function components, and that they are easier to read and test as they don't have to know about state or lifecycle hooks.